src/detect-engine-loader.c

```diff
@@ -77,23 +77,11 @@ char *DetectLoadCompleteSigPath(const DetectEngineCtx *de_ctx, const char *sig_f
 
     /* Path not specified */
     if (PathIsRelative(sig_file)) {
-        if (ConfGet(varname, &defaultpath) == 1) {
```

Comment:
should this be removed?

Reply:
doesn't look like defaultpath is initialized to anything other than NULL

Reply:
This commit is needed: 39570d8d0500a4a5fe8a9e9bc1aef406e8a7d4f4